web-client/src/presenter/computeds/Public/publicTrialSessionsHelper.ts

```diff
@@ -154,7 +182,12 @@ export const publicTrialSessionsHelper = (
 
   const groupedTrialSessions = groupTrialSessions(trialSessionRows);
 
+  const filteredGroups = groupedTrialSessions.filter(tsGroup =>
+    dateComparison(tsGroup),
+  );
+
   return {
+    filteredGroups,
     filtersHaveBeenModified,
     groupedTrialSessions,
```

Comment:
Since we no longer use `groupedTrialSessions` directly, we can stop exporting it/combine `filteredGroups` and `groupedTrialSessions`